models/balancer/ethereum/balancer_v1_ethereum_liquidity.sql

```diff
@@ -21,12 +22,12 @@ WITH prices AS (
         FROM {{ source('prices', 'usd') }}
         WHERE blockchain = 'ethereum'
         {% if is_incremental() %}
-        AND minute >= date_trunc("day", now() - interval '1 week')
+        AND minute >= date_trunc('day', now() - interval '7' day)
         {% endif %}
         GROUP BY 1, 2
     ),
     
-    dex_prices_1 AS (
+    /*dex_prices_1 AS (
```

Comment:
noticed here, but can you do another final sweep of commented out code that contains a `ref()` and use the jinja comments?

this bug only appears in prod

Reply:
So sorry, @jeff-dude. I guess it should be good now.